integration/broker-tests/tests-integration/tests-amqp/pom.xml

```diff
@@ -43,8 +43,8 @@
                     <argLine>-Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m</argLine>
                     <parallel>false</parallel>
                     <suiteXmlFiles>
-                        <suiteXmlFile>src/test/resources/testng-server-mgt.xml</suiteXmlFile>
-                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                        <!--<suiteXmlFile>src/test/resources/testng-server-mgt.xml</suiteXmlFile>-->
+                        <suiteXmlFile>${testng.suiteXml.location}</suiteXmlFile>
```

Comment:
Previously we had two profile files. But now we only have one. Is it a mistake?

Reply:
In the previous one ('testng-server-mgt.xml) we only startup the listeners without tests. And in the second(testng.xml) we start the test suite. I removed the listeners from the first one and added them to the testng.xml itself so that it's self contained. Will remove the commented line.

Reply:
Removed the line